academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Inter-Blockchain Communication Introduction"
+order: 1
+description: Introduction to the IBC Protocol
+tag: fast-track
+---
+
+# What is IBC?
+
+The **[(Inter-Blockchain Communication (IBC)](https://ibcprotocol.org/) Protocol** is _a protocol to handle authentication and transport of data between two blockchains_. IBC **requires a minimal set of functions**, specified in the [Interchain Standards (ICS)](https://github.com/cosmos/ibc/tree/master/spec/ics-001-ics-standard). Notice that those specifications do not limit the network topology or consensus algorithm, so IBC can be used with a wide range of blockchains or state machines. The IBC protocol provides a permissionless way for relaying data packets between blockchains, unlike most trusted bridging technologies. The security of IBC reduces to the security of the participating chains.
+
+IBC solves a widespread problem: cross-chain communication. This problem exists on public blockchains when exchanges wish to perform swaps. The problem arises early on in the case of application-specific blockchains where every asset is likely to emerge from its own purpose-built chain. Cross-chain communication is also a challenge in the world of private blockchains in cases where communication with a public chain or other private chains is desirable. There are already [IBC implementations for private blockchains such as Hyperledger Fabric and Corda](https://www.hyperledger.org/blog/2021/06/09/meet-yui-one-the-new-hyperledger-labs-projects-taking-on-cross-chain-and-off-chain-operations).
+
+Cross-chain communication in Cosmos enables parallelism and scaleability with transaction finality. This transaction finality solves well-known problems that plague other platforms: transaction costs, network capacity, and transaction confirmation finality.
+
+## Internet of blockchains
+
+IBC is essential for application-specific blockchains like the ones in the Cosmos network. It offers a standard communication channel for applications on two different chains that need to communicate with each other.
+
+Most Cosmos applications execute on their own purpose-built blockchain running their own validator set. These are the application-specific blockchains built with the Cosmos SDK. Applications on one chain may need to communicate with applications on another blockchain. For example, an application could accept tokens from another blockchain as a form of payment. Interoperability at this level calls for a method of exchanging data about the state or the transactions on another blockchain. While such bridges between blockchains can be built and do exist, they are generally constructed ad-hoc. IBC provides all Cosmos SDK applications with a common protocol and framework for implementing standardized inter-blockchain communication.
```

Comment:
"Most Cosmos applications execute on their own purpose-built blockchain running their own validator set." maybe mention upcoming Interchain security which could make this statement outdated soon?

"IBC provides all Cosmos SDK applications with a common protocol and framework for implementing standardized inter-blockchain communication."
You mention just a bit further IBC is not limited to SDK chains but I would still not formulate it that way here as well.
For example: "IBC provides chains with a common protocol and framework for implementing standardized inter-blockchain communication. With Cosmos SDK this comes out of the box, but also other types of chains that employ other consensus protocols or are built with different frameworks."